### ISSUE TITLE
support video play？

### ISSUE THREAD

**Issue by wiselike** [revel/revel#1553]
How to support MP4 or mp3 file play？

**Comment by brendensoares** [comment]
Can you be more clear? What are you trying to do? You want to serve mp3 files? Revel doesn't offer frontend features for audio playback, so...

**Comment by hengwu0** [comment]
> Can you be more clear? What are you trying to do? You want to serve mp3 files? Revel doesn't offer frontend features for audio playback, so...

Yes, i use revel to set up a blog website. And there are some pages will need media files to show.
I tried and it seems don't support video streaming serve for now.

**Comment by wiselike** [comment]
To serve mp4/mp3 files. Something like nginx_mod_h264_streaming, nginx-http-flv-module on nginx.

**Comment by wiselike** [comment]
Solved by nginx
